cli/src/main.rs

```diff
@@ -469,6 +476,16 @@ async fn main() -> Result<()> {
                 }
             }
         }
+        Command::Abandon { cluster, backends } => {
+            let cluster = ClusterName::new(&cluster);
+            for backend in backends {
```

Comment:
Given that this command is essentially a "trust me you can release this lock" command, I would prefer to put in a few safeguards here. I think the main one is that we should make sure we're only setting backends to Lost that aren't already in a terminal state. Doing that check first will also ensure that the backend definitely exists. Secondly, having a prompt that shows you some of the backend info and then asks you to confirm has been useful in the past. That's what the [set-backend-lost](https://github.com/drifting-in-space/p2/blob/main/jamsocket/src/admin/cli.rs#L266-L318) command in the admin CLI does right now, and this command should be able to replace that one.